src/main/resources/application.yml

```
@@ -1,3 +1,12 @@
-    name: backend
\ No newline at end of file
+    name: backend
+
+sentry:
+  dsn: ${SENTRY_DSN}
+  send-default-pii: true
+  traces-sample-rate: 1.0
+  profiles-sample-rate: 1.0
+  environment: development
+  debug: false
+  exception-resolver-order: 2147483647
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)